Jetcaster/app/build.gradle

```diff
@@ -88,7 +88,7 @@ android {
     }
 
     composeOptions {
-        kotlinCompilerExtensionVersion Libs.AndroidX.Compose.version
+        kotlinCompilerExtensionVersion = "1.2.0"
```

Comment:
should we create a compose compiler version in Libs.AndroidX.Compose so we don't hardcode the version of the compiler? 

Here and in the other projects as well

Reply:
Addressed in d5ff569a